lint-staged.config.js

```diff
@@ -10,6 +10,13 @@ function prettierMarkdown(filenames) {
 module.exports = {
   '{docs,samples}/**/*.{html,js,jsx,ts,tsx}': prettierCode,
   '**/*.md': prettierMarkdown,
-  'packages/{api,bundle,component,core,directlinespeech,playground}/src/**/*.{js,jsx,ts,tsx}': prettierCode,
-  '*.{js,jsx,ts,tsx}': 'npm run eslint'
+  'packages/acs-chat-adapter/src/**/*.{js,ts,tsx}': ['npm run precommit:eslint:acs-chat-adapter'],
+  'packages/api/src/**/*.{js,ts,tsx}': ['npm run precommit:eslint:api'],
+  'packages/bundle/src/**/*.{js,ts,tsx}': ['npm run precommit:eslint:bundle'],
+  'packages/component/src/**/*.{js,ts,tsx}': ['npm run precommit:eslint:component'],
+  'packages/core/src/**/*.{js,ts,tsx}': ['npm run precommit:eslint:core'],
+  'packages/directlinespeech/src/**/*.{js,ts,tsx}': ['npm run precommit:eslint:directlinespeech'],
+  'packages/isomorphic-react/src/**/*.{js,ts,tsx}': ['npm run precommit:eslint:isomorphic-react'],
```

Comment:
Didn't you say previously that you want to skip `isomorphic-react` and `isomorphic-react-dom` because we won't be making changes to those packages? Or am I remembering something else?